src/lib/components/AlignmentChart.react.js

```diff
@@ -35,6 +35,11 @@ export default class AlignmentChart extends Component {
     render() {
         const {id, eventDatum} = this.props;
 
+        // TODO handle this in react-alignment-viewer instead
+        if (this.props.colorscale === null) {
+            this.props.colorscale = 'clustal2';
+        }
```

Comment:
Playing with props this way makes me a bit queazy. Could be done alongside the omit

```
                {...{
                    ...omit(
                        ['fireEvent', 'dashEvent', 'setProps'],
                        this.props
                    )},
                    colorscale: this.props.colorscale === null ? 'clustal2': this.props.colorscale === null
                }}
```
or some variation

Reply:
@Marc-Andre-Rivet Would it be sufficient to just treat the `colorscale` prop as a boolean? e.g., 

`colorscale={this.props.colorscale ? this.props.colorscale : 'clustal2'}`

Reply:
This is fine if what you wanted to do was check truthiness not `null` specifically. This behaves differently with `undefined` and `0`.

Reply:
The behaviour I want is for the default to be `'clustal2'`, which means that it should revert to that whenever a colorscale isn't specified (which applies to `''`, `undefined`, and `null`). Please correct me if I'm wrong!